### PR TITLE
Fix news landing

### PR DESCRIPTION
**Pull request by mattandrews**
Spotted an issue where a lot of new posts in one category over another means there's a chance one category will be empty, eg:

image

This is also partly due to a bug in https://github.com/biglotteryfund/blf-alpha/pull/1527 which changes the format (eg. we now need to use `.result` for the entries, though I can't work out why we didn't need this before...).

### CODE REVIEW

**Comment by mattandrews** [comment]
Oh, and this also adds related programmes to press releases, which almost always have one (it turns out):

image

File: controllers/updates/index.js [hunk 1]
```diff
@@ -22,13 +22,22 @@ if (appData.isNotProduction) {
...
+                    locale: req.i18n.getLocale(),
+                    type: 'blog'
+                });
+
+                const allEntries = concat(pressReleases.result, blogposts.result);
+
                 res.render(path.resolve(__dirname, `./views/landing`), {
                     title: copy.title,
-                    groupedEntries: groupBy(response.result, 'entryType'),
+                    groupedEntries: groupBy(allEntries, 'entryType'),
```
Comment (davidrapson) [inline controllers/updates/index.js]: Do you need this groupBy anymore. Given you are concatenating the already grouped entries then ungrouping/grouping them again? Could you pass through the two separate results as they are?

File: controllers/updates/views/view-helpers.njk [hunk 2]
```diff
@@ -76,3 +76,26 @@
 
     {% endcall %}
 {% endmacro %}
+
+
+{% macro relatedProgrammes(entry) %}
```
Comment (davidrapson) [inline controllers/updates/views/view-helpers.njk]: Maybe make this into a first-class component in `components/`?

File: controllers/updates/index.js (hunk 1 above)
Comment (mattandrews) [reply controllers/updates/index.js]: Haha, great point. Will fix!

File: controllers/updates/views/view-helpers.njk (hunk 2 above)
Comment (mattandrews) [reply controllers/updates/views/view-helpers.njk]: Done.

**Pull request merged**